fakebook/src/app/component/profile-view/profile-view.component.html

```diff
@@ -14,6 +14,21 @@
   <h2 class="d-flex justify-content-center">
     {{ user?.firstName }} {{ user?.lastName }}
   </h2>
+  <div *ngIf="user.email !== currentUserEmail" class="col text-center">
+    <button 
+      type="button" 
+      class="btn text-center" 
+      [class]="followStatus ? 'btn-success' : 'btn-primary'" 
+      (click)="followUser()"
+    >
+      <div *ngIf="followStatus">
+        Following
+      </div>
+      <div *ngIf="!followStatus">
+        Follow
+      </div>
```

Comment:
divs do not belong inside buttons, it's invalid html

Reply:
changed to use ngContainer instead

Reply:
this would be a good time to use `else` with `ngIf` too but this is fine